tag.go

```diff
@@ -52,6 +54,9 @@ func (t Tag) String() string {
 			switch tb := b.(type) {
 			case htmler:
 				bb.Write([]byte(tb.HTML()))
+			case nulls.String:
```

Comment:
We shouldn't need this check. It should be caught by the "interfacer" check below it.